test_commands.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/sh
+# Test script for testing command entry points
+
+name="oval-graph"
+test_file_src="./tests/test_data/ssg-fedora-ds-arf.xml"
+tmp_dir_src="./tmp_data"
+tmp_json_file_src="${tmp_dir_src}/data.json"
+
+test_result=0
+overall_test_result=0
+
+usage() {
+    cat <<EOF
+usage: bash ./test_commands.sh
+-c    | --clean                  Removes created files
+-s    | --silent                 Disable every output of script
+-v    | --verbose                Displays details about the tests
+-io   | --install-oval-graph     Install oval-graph from source
+-h    | --help                   Brings up this menu
+EOF
+}
+
+clean=false
+verbose=false
+silent=false
+oval_graph_install=false
+
+for arg in "$@"; do
```

Comment:
I see that you parse command-line arguments manually. In your case, you could consider using a code generator to generate the help message and argument parsing code for you (e.g. [Argbash](https://argbash.readthedocs.org)), or you could consider using `getopts` to avoid the for loop.